test/development/basic/next-rs-api.test.ts

```diff
@@ -23,11 +24,38 @@ function normalizePath(path: string) {
     )
 }
 
+function styledStringToMarkdown(styled: StyledString): string {
+  switch (styled.type) {
+    case 'text':
+      return styled.value
+    case 'strong':
+      return '**' + styled.value + '**'
+    case 'code':
+      return '`' + styled.value + '`'
+    case 'line': {
+      let line = ''
+      for (const part of styled.value) {
+        line += styledStringToMarkdown(part)
+      }
+      return line + '\n'
```

Comment:
```suggestion
      return line
```

A line should not add a line break. That's the job of `stack`